Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -1519,7 +1519,7 @@ void NetworkConnectionToWebProcess::logOnBehalfOfWebContent(std::span<const char
     }
 
     auto osLogPointer = osLog.get() ? osLog.get() : OS_LOG_DEFAULT;
-    auto logData = byteCast<char>(logString.data());
+    auto* logData = logString.data();
```

Comment:
Should this be `logStringIncludingNullTerminator.data()`?

Reply:
Yes, this was fixed in a follow up.